### PR TITLE
runctl: allow cluster size to be set to CPUs

### PR DESCRIPTION
**Pull request by sam-github**
Command line and environment variable parsing supported a cluster size
of "cpus", but the set-size command did not, now it does.

### CODE REVIEW

File: test/test-runctl-clusterctl.js [hunk 1]
```diff
@@ -100,6 +102,14 @@ tap.test('runctl via clusterctl', function(t) {
   }, 'status worker id 6');
 
   t.doesNotThrow(function() {
+    expect('set-size CPUs');
```
Comment (rmg) [inline test/test-runctl-clusterctl.js]: Should do a `set-size 0` before this, otherwise the test isn't valid if it happens to run on a 2 core system (quite possible in CI environments).

**Comment by rmg** [comment]
Other than a corner case in the test assumptions, LGTM.

**Comment by sam-github** [comment]
Build failing for a real bug in 0.12. I'm fixing in https://github.com/strongloop/strong-cluster-control/pull/52, and will merge this because its needed for docker.
